test/runtests.jl

```diff
@@ -34,24 +34,40 @@ istest(f) = endswith(f, ".jl") && startswith(f, "test_")
 testfiles = sort(filter(istest, readdir(testdir)))
 
 @testset "$f" for f in testfiles
-    mpiexec() do cmd
+    mpiexec() do mpirun
+        cmd = `$mpirun -n $nprocs $(Base.julia_cmd()) $(joinpath(testdir, f))`
```

Comment:
```suggestion
        cmd(n=nprocs) = `$mpirun -n $n $(Base.julia_cmd()) $(joinpath(testdir, f))`
```

Reply:
Oh wow, that was so obvious in hindsight :smile:  Thanks!